UIforETW/Utility.cpp

```diff
@@ -67,27 +202,42 @@ std::vector<std::wstring> GetFileList(const std::wstring& pattern, bool fullPath
 // an embedded NUL then the resulting string will be truncated.
 std::wstring LoadFileAsText(const std::wstring& fileName)
 {
+	if (!::PathFileExistsW(fileName.c_str()))
+		return L"";
+
 	std::ifstream f;
 	f.open(fileName, std::ios_base::binary);
 	if (!f)
+	{
+		outputPrintf(L"Failed to load file `%s` as text - f.open failed!\n", fileName.c_str());

```

Comment:
Error messages should be aimed at users, not developers. f.open failed seems rather low-level. TMI I would say.